### DISCUSSION TITLE
403 Forbidden

### DISCUSSION THREAD

**Discussion by bluerabbitsu33** [AlistGo/alist#630] [Q&A]
使用rclone挂载alist webdav
复制其中文件到rclone其他挂载盘报错无法复制
rclone copy alist:/ali onedrive:/ -P
.7z: Failed to copy: failed to open source object: 403 Forbidden特别版
2022-02-27 23:42:40 ERROR : Attempt 1/3 failed with 1 errors and: failed to open source object: 403 Forbidden
.7z: Failed to copy: failed to open source object: 403 Forbidden特别版
2022-02-27 23:42:41 ERROR : Attempt 2/3 failed with 1 errors and: failed to open source object: 403 Forbidden
.7z: Failed to copy: failed to open source object: 403 Forbidden特别版
2022-02-27 23:42:42 ERROR : Attempt 3/3 failed with 1 errors and: failed to open source object: 403 Forbidden
Transferred:              0 B / 0 B, -, 0 B/s, ETA -
Errors:                 1 (retrying may help)
Elapsed time:         5.4s
2022/02/27 23:42:42 Failed to copy: failed to open source object: 403 Forbidden

**Comment by bluerabbitsu33** [comment]
补充
alist上挂载的是阿里云盘

**Comment by bluerabbitsu33** [comment]
补充
alist上挂载的是阿里云盘

**Reply by nitsort** [reply]
可以试试在 Rclone 加上`--header "Referer:"`

**Reply by zzwk** [reply]
> 可以试试在 Rclone 加上`--header "Referer:"`

完美解决！万分感谢！！确实想不到居然只加这几个字就能解决

**Reply by gzxiexl** [reply]
> 可以试试在 Rclone 加上`--header "Referer:"`

请教下你的rclone挂载命令能否分享一下

**Reply by nitsort** [reply]
> 

rclone mount --config rclone.conf Alist:/ Z: --network-mode --cache-dir cache --vfs-cache-mode full --header "Referer:"

**Reply by gzxiexl** [reply]
你的命令是win下用的吧？

**Reply by maryjeck** [reply]
rclone copy 怎么写啊，
我都是Failed to copy: failed to open source object: 403 Forbidden这个提示

**Reply by goooooouwa** [reply]
rclone config 添加header "Referer:" 后，上传文件报错：odd number of headers supplied。Referer不用赋值吗？

**Reply by euzenlee** [reply]
参数果然有效，解决问题了。

**Reply by aTenb** [reply]
以前怎么没遇到这个问题，大佬1年前就遇到了，我最近才遇到，好奇怪

**Reply by Vincistudio** [reply]
大佬 请问我用truenas碰到同样的问题，这段代码是不是要进rclone config去修改 另外这个是更新什么的会丢失吗？

**Reply by BlueSkyXN** [reply]
> 大佬 请问我用truenas碰到同样的问题，这段代码是不是要进rclone config去修改 另外这个是更新什么的会丢失吗？

存储的具体配置->webdav配置->本地代理而不是302

**Reply by Vincistudio** [reply]
https://github.com/alist-org/alist/discussions/630#discussioncomment-7517577
肥肠感谢 如果是302就必须要赋予值吗？

**Reply by afghn** [reply]
> 可以试试在 Rclone 加上`--header "Referer:"`

man,what can i say,you the hero

**Reply by xuecl2** [reply]
> 可以试试在 Rclone 加上`--header "Referer:"`

非常感谢，看起来是rclone 重定向的时候出了问题，感觉这明明应该是个很普遍的问题，不知道为什么很少有人提

**Reply by puppetdevz** [reply]
WebDAV好像不支持302转发，只能改为本地代理。加上 `--header "Referer:"` 也不行。

**Comment by WHY12312311** [comment]
解决了，感谢感谢
